docs/tutorial.rst

```diff
@@ -204,41 +206,82 @@ b) Writing a gain calibration file.
       dd = []
       fl = []
       ch = []
-      for ant in antenna_numbers:
+      for ant in ant_array:
           dd.append(gains[pol][ant])
```

Comment:
I'm on board with creating random arrays with the correct shapes from the get go.